airbyte-migration/src/main/resources/migrations/migrationV0_19_0/Notification.yaml

```diff
@@ -0,0 +1,20 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardWorkspaceConfiguration.yaml
+title: Notification
+description: Notification Settings
+type: object
+required:
+  - notificationType
+additionalProperties: false
+properties:
+  # Instead of this type field, we would prefer a json schema "oneOf" but unfortunately,
+  # the jsonschema2pojo does not seem to support it yet: https://github.com/joelittlejohn/jsonschema2pojo/issues/392
+  notificationType:
+    "$ref": NotificationType.yaml
+  webhook:
+    type: string
+  onFailure:
```

Comment:
this file is out of sync with what's in `airbyte-config/models/src/main/resources/types/Notification.yaml`. or maybe just has the wrong name?

Reply:
not sure. something isn't right here.

Reply:
yes thanks, with the back and forth, I keep forgetting the migration part too